datacenter/ucp/2.1/guides/admin/configure/content-trust/admin_tasks.md

```diff
@@ -0,0 +1,387 @@
+    Enter username: admin
+    Enter password:
+    ```
+
+    As the help text in the command says, it's important to choose good
+    passphrases and to save them in a secure location such as a password
+    manager. The final username and password prompt are for the DTR login.
+
+    Several important files are saved in the trust directory (the location you
+    specified as the value of the `-d` flag. The following is an example listing
+    of the trust directory:
+
+    ```none
+    ├── private
+    │   ├── root_keys
+    │   │   └── 92c11d487023de4447ef57747e84e7364cd7c62a4be28d8714ec05afe2f130f8.key
+    │   └── tuf_keys
+    │       └── dtr-example.com
+    │           └── engineering
+    │               └── testrepo
+    │                   ├── 87a129ea47a4112fec6b989bde35f6ddea8325638450d41b3f44fabaf49dbe3d.key
+    │                   └── aa4b236c610e3d951c930bf2a503861c41808ac28369bfbaf5c075b62cb3dd41.key
+    └── tuf
+        └── dtr-example.com
+            └── engineering
+                └── testrepo
+                    ├── changelist
+                    └── metadata
+                        ├── root.json
+                        ├── snapshot.json
+                        └── targets.json
+    ```
+
+    The `tuf` directory contains metadata needed by Notary. The `private`
+    directory contains the root key, target key, and snapshots key. It is
+    important to protect these keys, especially the root key. If you are using
+    the Yubikey integration feature, the root key is already stored on your
+    Yubikey. You should back up the entire `private` subdirectory to secure
+    offline storage and remove the `root_keys` subdirectory from the trust
+    directory. If you do not use a Yubikey, back up the entire trust
+    directory to secure offline storage, and bring it online only when you need
+    to perform Notary operations.
+
+3.  The metadata has been created but only exists on your client. To publish
+    it to DTR, use `notary publish`.
+
+    ```bash
+    $ notary publish <GUN>
+
+    Pushing changes to <GUN>
+    Enter username: admin
+    Enter password:
+    Enter passphrase for targets key with ID 63c2d66:
+    Enter passphrase for snapshot key with ID 6ac388d:
+    Successfully published changes for repository <GUN>
+    ```
+
+    You will be prompted for the DTR login, the passphrase for the `targets` key,
+    and the passphrase for the `snapshot` key.
+
+Typically, the administrator is not part of the group which is authorized to
+sign images. If you do attempt to sign images and you are not part of one of the
+correct groups, the image will not be available to UCP.
+
+Continue to [delegate image signing](#delegate-image-signing) to give the
+appropriate users the ability to sign images.
+
+You can also
+[learn more about the keys used by Notary](/engine/security/trust/trust_key_mng.md).
+
+
+## Delegate image signing
+
+The administrator who manages Docker Trusted Registry is often not part of the
```

Comment:
Even if they are a member of a group allowed to sign images, they probably still have to add their UCP bundle key to a delegation - the targets key or root key is not recognized by UCP as a signing user.